src/commonMethods/commonFunctions.js

```diff
@@ -5,6 +5,8 @@ import { Client } from 'ssh2'
 
 import URL from 'url'
 
+let hostIP
```

Comment:
Since you call `deviceIP.call(this)` in every function anyway, I don't think we need this global variable.